Makefile

```diff
@@ -116,6 +116,13 @@ dockerbuild: ## Runs docker build
 	docker build . --file Dockerfile --tag $(IMAGE_NAME) 
 	docker build . --file Dockerfile.gsutil --tag $(IMAGE_NAME)-gsutil
 
+.PHONY: checkprojects
 checkprojects: ## Validates ./cron/projects.txt
 	cd ./scripts && go build  -o validate
 	./scripts/validate ./cron/projects.txt
+
+.PHONY: build-update
+build-update: ## builds the scripts/update binary
+	cd ./scripts/update && go mod tidy && go mod verify	
+	cd ./scripts/update && CGO=0 go build .
+	./scripts/tree-status
```

Comment:
build-update name is little confusing, this seems like projects-update ?
also, above checkprojects should be check-projects (dash in between).

Reply:
Agreed, took care of it. Thanks